### PR TITLE
Initial application setup

### PR DESCRIPTION
**Pull request by ddohler**
Initialize DRIVER app using ansible roles migrated from ASHLAR.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+*.doctree
+*.swp
+*.swo
+*.pyc
+*.suo
+*.user
+*.sql3
+
+# Alembic
+alembic.ini
```
Comment (CloudNiner) [inline .gitignore]: Oops, missed this forever ago. Can be deleted.

File: Vagrantfile [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+        # Mount options causing some users trouble, disable until necessary (e.g. a grunt/gulp install)
+        #app.vm.synced_folder "./app", "/opt/app", :nfs => true, :mount_options => [
+            #("nfsvers=3" if ENV.fetch("ASHLAR_NFS_VERSION_3", false)),
+            #"noatime",
+            #"actimeo=1",
+        #]
+    end
+
+    # nginx
+    app.vm.network "forwarded_port", guest: 80, host: Integer(ENV.fetch("ASHLAR_WEB_PORT_80", 7000))
```
Comment (ddohler) [inline Vagrantfile]: Good call, done.

**Comment by kshepard** [comment]
+1

**Comment by CloudNiner** [comment]
Merging so I can being work on this.

**Pull request merged**